src/command/dev/router/command.rs

```diff
@@ -39,6 +40,13 @@ impl BackgroundTask {
 
         command.stdout(Stdio::piped()).stderr(Stdio::piped());
 
+        if let Ok(apollo_key) = var("APOLLO_KEY") {
+            command.env("APOLLO_KEY", apollo_key);
```

Comment:
i assumed that child processes would inherit all env variables - is that not the case? might be nice to inject the api key from `--profile` if `APOLLO_KEY` is not set 🤷🏼

Reply:
Apparently, it's not since this wasn't working 😓. I think `Command::run` _does_ inherit, which maybe works since we're spawning this in another thread? But who knows what else that might mess up.

`--profile` is a good idea, maybe as a follow-on, though, since I'll have to poke around to find out how it works 😅